playbooks/post-config.yaml

```diff
@@ -43,3 +43,7 @@
         - zuul-merger
         - zuul-fingergw
         - zuul-web
+
+- name: Create Labsync task
+  import_playbook: sync-config-files.yaml
```

Comment:
not here, I mean you can put this `import_playbook ...` in site.yaml, but before the `import_playbook: post_config.yaml`

Reply:
e.... ok, will change this.